game.rb

```diff
@@ -24,21 +23,28 @@ def player_one_pick_word
   end
 
   def player_two_guess_word word
-    counter = 0
+    guess_counter = 0
```

Comment:
:+1: Nothing like some better variable names. This should probably become `incorrect_guess_counter` though I imagine?